tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -140,6 +142,47 @@ function get_container_runtime() {
 	fi
 }
 
+function is_containerd_capable_of_using_drop_in_files() {
```

Comment:
I'm trying to decide if `is_container_runtime_capable_of_using_drop_in_files` is a better name for this as it has take runtimes other than containerd, but it's testing a containerd feature, so I think I'm probably bike shedding and I think it's fine as it is.

Reply:
I will leave it as it is for now, as CRI-O is capable of using drop-in files, but we approach that in a different way.